src/tox_extra/hooks.py

```diff
@@ -1,34 +1,54 @@
 """Tox hook implementations."""
```

Comment:
We should update the setup.cfg to require >=4 no?

Reply:
This is keeping the compatibility with v3, so I would not bump it yet. Another reason to not do it is that v4 is not released yet and adding condition on pre-release libraries is bad, can cause pip to install other pre-release libraries too.

But I do plan to add that line relatively soon after v4 goes out.

Now the v4 compatibility is tested by the `-devel` jobs.